app/client/views/settings/users/enroll/quickfromDropdown.html

```diff
@@ -0,0 +1,12 @@
+<template name="quickForm_dropdown">
+  <div class="form-group">
+    <label for="">
+      {{_ "editUserGroups-groupsLabel" }}
+    </label>
+    <select multiple data-schema-key {{atts}} class="form-control" name="groups">
+      {{#each selectOptions}}
```

Comment:
Just pass selectOptions into the existing User Groups select widget. If the select options use the same structure, then the pre-existing widget can be re-used here.

Reply:
Here is what I am aiming for:

```html
{{> userGroupsSelectWidget groupSelectOptions=allGroups }}
```